docker-compose.yaml

```
@@ -0,0 +1,13 @@
+version: "3.3"
+
+services:
+
+  site:
+    image: servicenow/github-pages-hugo
```

We can use a more generic name for the image.

```suggestion
    image: docsy/docsy-example
```

---

I will update the PR!